traits/util/trait_documenter.py

```diff
@@ -129,32 +145,65 @@ def add_directive_header(self, sig):
 
         self.add_line("   :annotation: = {0}".format(definition), "<autodoc>")
 
-    # Private Interface #####################################################
 
-    def _get_trait_definition(self):
-        """ Retrieve the Trait attribute definition
-        """
+def trait_definition(*, cls, trait_name):
```

Comment:
ref : keyword only arguments have been available in python 3 since python 3.0 - https://docs.python.org/3/whatsnew/3.0.html#new-syntax. ref https://www.python.org/dev/peps/pep-3102

Reply:
That's true, but I don't understand why it's worthy of comment. Is there something that you think needs to be changed here?

Reply:
nope nope just for personal record (probably shouldn't use github PR review comments for personal record)